src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/ApplicationTestCase.kt

```diff
@@ -314,4 +343,10 @@ abstract class ApplicationTestCase {
     whenever(dateTimeProvider.now)
       .thenAnswer { Optional.of(startTime.plus(Period.ofDays(this.countOfGettingCurrentTime[0]++))) }
   }
+
+  protected fun randomAlphabets(length: Int): String =
```

Comment:
As part of improving our testing strategy, we could add a task to our TODO list to investigate a tool like https://github.com/DiUS/java-faker to provide curated, realistic random data in our tests.

I think that is not a priority now; first, we should refactor our test data in proper mother objects. But this is a good starting point :-)

Reply:
raised a TODO card to dev backlog, with two items now (parent objects, faker/stub)